docs/sources/reference/cli/validate.md

```diff
@@ -0,0 +1,40 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/cli/validate/
+description: Learn about the validate command
+menuTitle: validate
+title: The validate command
+weight: 200
+---
+
+# The `validate` command
+
+The `validate` command validates {{< param "PRODUCT_NAME" >}} configuration file/directory path.
+
+## Usage
```

Comment:
Can you describe how to use it in CI programmatically, e.g. the non-zero exit code = error, zero exit code = success.

Reply:
[2a1128d](https://github.com/grafana/alloy/pull/3220/commits/2a1128d1ffc9cb3b6494ca8c01acc9fb1a4dbcd0)